packages/design-system/src/Switch/Switch.types.ts

```diff
@@ -8,4 +8,5 @@ export type SwitchProps = {
   isSelected?: boolean;
   isDisabled?: boolean;
   children?: ReactNode;
+  onChange?: () => void;
```

Comment:
signature of onChange is wrong here. You can revert this change since I have already fixed this in another PR.
https://github.com/appsmithorg/design-system/pull/444